src/main/java/com/yahoo/bullet/querying/Querier.java

```diff
@@ -665,6 +671,9 @@ private Clip postAggregate(Clip clip) {
                 clip = postStrategy.execute(clip);
             }
         }
+        for (String field : transientFields.keySet()) {
```

Comment:
transientFields.keySet().stream().forEach(record::remove)

Reply:
Also move this inside the the postStrategies != null check

Reply:
Maybe invert the the null check and return instead so that this all doesn't need to be nested

Reply:
I can not use stream because here we have 2 nested loops: each field, each record